website/src/pages/docs/features/introspection.mdx

```diff
@@ -68,6 +68,31 @@ server.listen(4000, () => {
 })
 ```
 
+## Disable Introspection based on the context
+
+You can also disable introspection based on the context. This is useful when you want to disable it
+via another auth plugin like [JWT Plugin](/docs/features/jwt).
+
+```ts "Disabling GraphQL schema introspection based on the context" {7}
+import { createYoga } from 'graphql-yoga'
+import { useDisableIntrospection } from '@graphql-yoga/plugin-disable-introspection'
+
+// Provide your schema
+const yoga = createYoga({
+  graphiql: false,
+  plugins: [
+    useDisableIntrospection({
+      isDisabled: (_req, ctx) => !ctx.jwt
+    })
+  ]
+})
```

Comment:
Maybe also add the JWT plugin since we're mentioning it.

```suggestion
import { createYoga } from 'graphql-yoga'
import { useJWT } from '@graphql-yoga/plugin-jwt';
import { useDisableIntrospection } from '@graphql-yoga/plugin-disable-introspection'

// Provide your schema
const yoga = createYoga({
  graphiql: false,
  plugins: [
    useJWT(),
    useDisableIntrospection({
      isDisabled: (_req, ctx) => !ctx.jwt
    })
  ]
})
```

Reply:
Makes sense!